grand_challenge_forge/forge.py

```diff
@@ -266,7 +268,7 @@ def generate_algorithm_template(
         "grand-challenge-forge"
     )
 
-    algorithm_slug = context["algorithm"]["slug"]
+    algorithm_slug = context["algorithm"]["slug"].lower()
```

Comment:
Just be careful with reusing this - if you use that as the slug in any API code it will fail as the lookups are case sensitive. Just using it for the template path is okay. May be worth lowering just where it is used instead:

```python
template_path = output_path / f"{algorithm_slug.lower()}-template"
```